docs/product-assembly_pam-on-openshift/src/main/asciidoc/ba-openshift-overview-con.adoc

```diff
@@ -0,0 +1,35 @@
+[id='ba-openshift-overview-con']
+= Overview
+If you have an OpenShift environment, you can deploy {PRODUCT} into this environment.
+
+In this solution, components of {PRODUCT} are deployed as separate OpenShift pods. You can scale each of the pods up and down individually, providing as few or as many containers as necessary for a particular component. You can use standard OpenShift methods to manage the pods and balance the load.
+
+The following key components are available as pods running in OpenShift:
+
+* {KIE_SERVER}, also known as _KIE Server_, is the infrastructure element that runs a process or a group of processes. All process logic runs on execution servers.
++
+A database backend is required for an execution server. You can provide a database server in another OpenShift pod or configure an execution server on OpenShift to use any other database server.
++
+You can freely scale up an {KIE_SERVER} pod, providing as many copies, running on the same host or different hosts, as necessary. As you scale a pod up or down, all its copies run the same processes and use the same database backend. OpenShift provides load balancing and a request can be handled by any of the pods.
```

Comment:
scale up **a** {KIE_SERVER} pod

Reply:
_OpenShift provides load balancing_: I would need to double check, but I think there is some nuance to this and it may not always be true. It is true if your application calls the KIE server through its service hostname (which you should be), but I think the smart router directly calls individual pod IPs, thereby acting as the load balancer and skipping OCP load balancing.